tasks/setup-RedHat.yml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: Ensure NFS utilities are installed.
-  yum: name=nfs-utils state=installed
+  package: name=nfs-utils state=installed
+
+- name: Ensure libselinux-python is available on Fedora
```

Comment:
I believe this library would be a prerequisite for many different modules Ansible uses; it would typically be installed outside of an individual role (since I think it's also needed for `file`, `copy` and other modules). I'd rather leave this task out (or consider it in a separate PR) so just the package stuff is fixed for now.